packages/okta-angular/README.md

```diff
@@ -83,9 +83,38 @@ Routes are protected by the `OktaAuthGuard`, which verifies there is a valid `ac
   component: MyProtectedComponent,
   canActivate: [ OktaAuthGuard ]
 }
-``` 
+```
+
+By default, the `OktaAuthGuard` will redirect the user to Okta for authentication. You can modify this functionality by adding a `data` attribute:
```

Comment:
We should move all of this custom-login-page content under a new heading "Using a Custom Login Page", this section should come before "Development".  The reasoning is (1) taking about the custom login page is a distraction from the "let's get the default case working" flow and (2) we need to elaborate on this custom login page case, as written this leaves them hanging.  We need to tell them that a custom login pages is intended to be used with the sign-in widget, and that they are responsible for using the widget to kick off the implicit redirect flow.  Keep this terse and link to SIW docs for more details.  After samples V2 strategy is complete we will also have a concrete example to link to :)

Reply:
I wanted to hold off on adding explicit detail about a **custom login** page, since we will be updating our **[Okta + Angular + Sign-In Widget](https://okta.github.io/code/angular/okta_angular_sign-in_widget.html)** example to explain how this works.

Additionally, we have the problem of duplication in this README. It is nearly identical to the quickstart. As you mentioned, we should point the reader to our samples, quickstarts, and walkthroughs to avoid this.